### PR TITLE
Fix CVEs in kommander ui.

### PR DESCRIPTION
**Pull request by chhsia0**

### CODE REVIEW

**Review by jbarrick-mesosphere — DISMISSED** [review]

File: addons/dispatch/1.4.x/dispatch-3.yaml [hunk 1]
```diff
@@ -85,6 +85,8 @@ spec:
       dispatch:
         image: "mesosphere/dispatch:1.4.0"
         testImage: "mesosphere/dispatch-helm-test:1.4.0"
+      kommander-ui:
+        defaultTag: 6.84.2
```
Comment (TattdCodeMonkey) [inline addons/dispatch/1.4.x/dispatch-3.yaml]: the usage is `:{{ default (tpl .Values.image.defaultTag .) .Values.image.tag }}` in the [kommander-ui chart](https://github.com/mesosphere/kommander-ui/blob/master/chart/kommander-ui/templates/deployment.yaml#L41). 

Should this be:
```
kommander-ui:
   image:
      defaultTag: 6.84.2
```

Comment (chhsia0) [reply addons/dispatch/1.4.x/dispatch-3.yaml]: oops. thanks.

**Review by gracedo — APPROVED** [review]
LGTM, thanks! Can we also queue up a PR to update the version in charts for post-release?

**Comment by chhsia0** [comment]
Sure will do!

**Pull request merged**